src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/4.diff.sql

```diff
@@ -0,0 +1,78 @@
+/*************************************************************
+    Search Parameter Status Information
+**************************************************************/
+
+ALTER TABLE dbo.SearchParam
+ADD
+    Status varchar(10) NOT NULL,
+    LastUpdated datetimeoffset(7) NOT NULL,
+    IsPartiallySupported bit NOT NULL
+
+CREATE TYPE dbo.SearchParamTableType_1 AS TABLE
```

Comment:
@johnstairs I remember you saying these are immutable, is this ok or is there another versioning strategy we should consider? `_1` vs `v2`?